googledatastore/pom.xml

```diff
@@ -29,11 +29,21 @@ LICENSE file.
   <name>Google Cloud Datastore Binding</name>
   <url>https://github.com/GoogleCloudPlatform/google-cloud-datastore</url>
 
+  <repositories>
+    <repository>
+      <id>sonatype-snapshots</id>
+      <name>sonatype-snapshots</name>
+      <url>https://oss.sonatype.org/content/repositories/snapshots/</url>
```

Comment:
do we still need this snapshot repository? I think the client is in maven central.